apps/reference/_gotrue/usage.mdx

```diff
@@ -0,0 +1,408 @@
+}
+```
+
+Returns:
+
+```json
+{
+  "id": "11111111-2222-3333-4444-5555555555555",
+  "aud": "authenticated",
+  "role": "authenticated",
+  "email": "email@example.com",
+  "app_metadata": {
+    "provider": "email"
+  },
+  "user_metadata": null,
+  "created_at": "2016-05-15T19:53:12.368652374-07:00",
+  "updated_at": "2016-05-15T19:53:12.368652374-07:00"
+}
+```
+
+### **GET /admin/users/{user_id}**
+
+Gets a user. Requires your `service_role` API key and thus should only be used
+in secure server-side environments.
+
+Returns:
+
+```json
+{
+  "id": "11111111-2222-3333-4444-5555555555555",
+  "aud": "authenticated",
+  "role": "authenticated",
+  "email": "email@example.com",
+  "app_metadata": {
+    "provider": "email"
+  },
+  "user_metadata": {},
+  "created_at": "2016-05-15T19:53:12.368652374-07:00",
+  "updated_at": "2016-05-15T19:53:12.368652374-07:00"
+}
+```
+
+### **PUT /admin/users/{user_id}**
+
+Updates a user. Requires your `service_role` API key and thus should only be
+used in secure server-side environments.
+
+```json
+{
+  "email": "email@example.com",
+  "password": "updated-password",
+  "data": {
+    "key": "updated-value",
+    "number": 10,
+    "admin": false
+  }
+}
+```
+
+Returns:
+
+```json
+{
+  "id": "11111111-2222-3333-4444-5555555555555",
+  "aud": "authenticated",
+  "role": "authenticated",
+  "email": "email@example.com",
+  "app_metadata": {
+    "provider": "email"
+  },
+  "user_metadata": {},
+  "created_at": "2016-05-15T19:53:12.368652374-07:00",
+  "updated_at": "2016-05-15T19:53:12.368652374-07:00"
+}
+```
+
+### **POST /admin/generate_link**
+
+Returns the corresponding email action link based on the type specified. The response also contains the query params of the action link as separate JSON fields for convenience (along with the email OTP from which the corresponding token is generated).
+
+Returns:
+
+```js
+{
+  "action_link": "http://localhost:9999/verify?token=TOKEN&type=TYPE&redirect_to=REDIRECT_URL",
+  "email_otp": "EMAIL_OTP",
+  "hashed_token": "TOKEN",
+  "verification_type": "TYPE",
+  "redirect_to": "REDIRECT_URL",
+  ...
+}
+```
+
+### **DELETE /admin/users/{user_id}**
+
+Deletes a user. Requires your `service_role` API key and thus should only be
+used in secure server-side environments.
+
+### **POST /logout**
+
+Logout a user (Requires authentication).
+
+This will revoke all refresh tokens for the user. Remember that the JWT tokens
+will still be valid for stateless auth until they expires.
+
+### **GET /authorize**
+
+Get access_token from external oauth provider
+
+query params:
+
+```
+provider=google | bitbucket | github | gitlab
+```
+
+Redirects to provider and then to `/callback`
+
+### **GET /callback**
+
+External provider should redirect to here
+
+Redirects to `<GOTRUE_SITE_URL>#access_token=<access_token>&refresh_token=<refresh_token>&expires_in=3600&provider=<provider_name>`
+
+## Pre-built
+
+- [Docker](https://hub.docker.com/repository/docker/supabase/gotrue)
```

Comment:
Do we still need this? We should add a few sentences for how to use it if we're keeping it (currently links to a login page).

Reply:
Think users need to sign up for a DockerHub account and log in in order to access the page. Thereafter, they'd need to run `docker pull supabase/gotrue:latest` and then `docker run -it gotrue` to get GoTrue up and running.

Reply:
@dannykng yup it looks great there